### PR TITLE
fix: refactor all buttons (sizes, colors), fix welcome block for narrow body, refactor signin and onboarding, remove garbage

### PR DESCRIPTION
**Pull request by iqmulator**

### CODE REVIEW

File: src/dotnet/Chat.UI.Blazor/Components/Banners/AddChatMembersBanner.razor [hunk 1]
```diff
@@ -10,10 +10,10 @@
     Dismiss="OnDismiss">
     <Icon>@* Intentionally empty *@</Icon>
     <Body>
-        <Button Class="btn-transparent" Click="OnInviteClick">
-            <i class="text-2xl icon-person-add"></i>
-            Add members
-        </Button>
+    <Button Class="btn-transparent unhovered" Click="OnInviteClick">
+        <IconContent><i class="text-xl icon-person-add"></i></IconContent>
```
Comment (alexis-kochetov) [inline src/dotnet/Chat.UI.Blazor/Components/Banners/AddChatMembersBanner.razor]: Can we rename to  `Icon` and `Title` ?

File: src/dotnet/UI.Blazor/Components/Button/Button.razor [hunk 2]
```diff
@@ -25,6 +38,8 @@
     [Parameter] public string Tooltip { get; set; } = "";
     [Parameter] public FloatingPosition TooltipPosition { get; set; }
     [Parameter] public RenderFragment? ChildContent { get; set; }
+    [Parameter] public RenderFragment? IconContent { get; set; }
```
Comment (alexis-kochetov) [inline src/dotnet/UI.Blazor/Components/Button/Button.razor]: Let's rename to `Title` and `Icon`

**Review by alexis-kochetov — CHANGES_REQUESTED** [review]

File: src/dotnet/Chat.UI.Blazor/Components/Banners/AddChatMembersBanner.razor (hunk 1 above)
Comment (iqmulator) [reply src/dotnet/Chat.UI.Blazor/Components/Banners/AddChatMembersBanner.razor]: Yes, we can

File: src/dotnet/UI.Blazor/Components/Button/Button.razor (hunk 2 above)
Comment (iqmulator) [reply src/dotnet/UI.Blazor/Components/Button/Button.razor]: Ok

**Review by alexyakunin — APPROVED** [review]

**Review by alexis-kochetov — APPROVED** [review]

**Pull request merged**